### PR TITLE
Raises Loadout Points to 15

### PR DESCRIPTION
**Pull request by ParadoxSpace**
As title. This shouldn't really harm anything.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
!review

**Comment by AmoryBlaine** [comment]
This is a fine temp fix. But we really need to just remove points all together for most accessories.

**Comment by Geevies** [comment]
Can't say I'm a fan of adding more loadout points. I suggest lowering the point cost of some non-overlapping accessories instead.

**Comment by MarinaGryphon** [comment]
> Can't say I'm a fan of adding more loadout points. I suggest lowering the point cost of some non-overlapping accessories instead.

so lower them to zero, which effectively removes the point limit altogether? reduce them to 0.5, potentially cause issues with decimals, and basically just double the point capacity while also doubling the relative price of more expensive accessories?

they are literally the same thing lmao

File: html/changelogs/paradoxspace-bloat.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+################################
+# Example Changelog File
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+#
+# Note: This file, and files beginning with ".", and files that don't end in ".yml" will not be read. If you change this file, you will look really dumb.
+#
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+# Your changelog will be merged with a master changelog. (New stuff added only, and only on the date entry for the day it was merged.)
+# When it is, any changes listed below will disappear.
+#
+# Valid Prefixes:
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# Valid Prefixes: 
+#   bugfix
+#   wip (For works in progress)
+#   tweak
+#   soundadd
+#   sounddel
+#   rscadd (general adding of nice things)
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+#   maptweak
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+#   spellcheck (typo fixes)
+#   experiment
+#   balance
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+#   admin
+#   backend
+#   security
+#   refactor
+#################################
+
+# Your name.  
+author: ParadoxSpace
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

File: html/changelogs/paradoxspace-bloat.yml [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
```
Comment (VTCobaltblood) [inline html/changelogs/paradoxspace-bloat.yml]: Nice

**Comment by Arrow768** [comment]
It´s fine as a temporary fix, until the loadout system can be reworked.

**Pull request merged**